### PR TITLE
RATEST-213: Explicitly import pages than using wildcard

### PR DESCRIPTION
**Pull request by mherman22**
This PR addresses --> [ticket](https://issues.openmrs.org/browse/RATEST-213) in trying to replace importation of pages using wild cards with importing of specific pages to be used in **Delete Request Appointment and Add Patient Appointment Tests**  and also remove some unnecessary spaces in the test classes

### CODE REVIEW

**Comment by HerbertYiga** [comment]
> @mherman22 what do you think of this empty code line [here](https://github.com/mherman22/openmrs-distro-referenceapplication/blob/e0b6391f22597ee6083cca4a7469b6d07a6010e3/ui-tests/src/test/java/org/openmrs/reference/DeleteRequestAppointmentTest.java#L30) ?

thanks @mherman22 ,  @kdaud i think the spacing is ok , it looks good to be merged in @dkayiwa

**Review by HerbertYiga — APPROVED** [review]

**Comment by kdaud** [comment]
@mherman22 did you take a look at the ticket description ?

Also add a description to you commit to provide a glimpse of what the PR is addressing!

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 1]
```diff
@@ -16,15 +16,18 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.groups.BuildTests;
-import org.openmrs.reference.page.*;
+import org.openmrs.reference.page.AppointmentSchedulingPage;
+import org.openmrs.reference.page. ManageProviderSchedulesPage;
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:20]: @mherman22 worth taking a look on the import package at `code line 20`!

**Comment by kdaud** [comment]
@mherman22 its always the role of the reviewer to make his/her resolved not the PR author! I personally find it helpful to leave reviews open for any preceding reviewer to be  able see what  has been reviewed already!

**Comment by mherman22** [comment]
> @mherman22 its always the role of the reviewer to mark his/her reviews resolved not by the `PR-author`! I personally find it helpful to leave reviews open for any preceding reviewer to be able see what has been reviewed already to lessen time spent reviewing a PR!

:+1:

**Comment by mherman22** [comment]
@kdaud can you please guide me on how to get these failed tests working. i wanna learn from this particular issue arising out of the testfailures which seem to be arising from EndVisitTest

**Comment by kdaud** [comment]
> guide me on how to get these failed tests working. i wanna learn from this particular issue arising out of the testfailures which seem to be arising from EndVisitTest

Are you getting the tests failures when building the module locally ?

**Comment by mherman22** [comment]
> > guide me on how to get these failed tests working. i wanna learn from this particular issue arising out of the testfailures which seem to be arising from EndVisitTest
> 
> Are you getting the test failures when building the module locally ?

i see they the tests have passed here, however locally when i run **mvn clean install i get** ```
Compilation failure
[ERROR] No compiler is provided in this environment. Perhaps you are running on a JRE rather than a JDK?

```

**Comment by kdaud** [comment]
@mherman22 have you juts forgotten to share the full log report via https://pastebin.com/ ?

**Comment by mherman22** [comment]
> provided

https://pastebin.com/8gk576yf

**Comment by mherman22** [comment]
> > Compilation failure
> > [ERROR] No compiler is provided in this environment. Perhaps you are running on a JRE rather than a JDK?
> 
> @mherman22 have you just forgotten to share the full log report via https://pastebin.com/ ?

sorry for that, clicked wrong button. anyway here is the log ---> https://pastebin.com/8gk576yf

**Comment by kdaud** [comment]
Which output do you get from the command  `javac -version `?

**Comment by mherman22** [comment]
> javac -version

Command 'javac' not found, but can be installed with:

sudo apt install openjdk-11-jdk-headless  # version 11.0.11+9-0ubuntu2~20.04, or
sudo apt install default-jdk              # version 2:1.11-72
sudo apt install openjdk-16-jdk-headless  # version 16.0.1+9-1~20.04
sudo apt install openjdk-8-jdk-headless   # version 8u292-b10-0ubuntu1~20.04
sudo apt install openjdk-13-jdk-headless  # version 13.0.7+5-0ubuntu1~20.04
sudo apt install ecj                      # version 3.16.0-1

**Comment by kdaud** [comment]
> > javac -version
> 
> Command 'javac' not found, but can be installed with:
> 
> sudo apt install openjdk-11-jdk-headless # version 11.0.11+9-0ubuntu2~20.04, or sudo apt install default-jdk # version 2:1.11-72 sudo apt install openjdk-16-jdk-headless # version 16.0.1+9-1~20.04
> sudo apt install openjdk-8-jdk-headless # version 8u292-b10-0ubuntu1~20.04 sudo apt install openjdk-13-jdk-headless # version 13.0.7+5-0ubuntu1~20.04
> sudo apt install ecj # version 3.16.0-1

It means that you are missing java compiler that is responsible for run maven builds.

Can you share the output of `java -version `?

**Comment by mherman22** [comment]
> > > javac -version
> > 
> > 
> > Command 'javac' not found, but can be installed with:
> > sudo apt install openjdk-11-jdk-headless # version 11.0.11+9-0ubuntu2~20.04, or sudo apt install default-jdk # version 2:1.11-72 sudo apt install openjdk-16-jdk-headless # version 16.0.1+9-1~20.04
> > sudo apt install openjdk-8-jdk-headless # version 8u292-b10-0ubuntu1~20.04 sudo apt install openjdk-13-jdk-headless # version 13.0.7+5-0ubuntu1~20.04
> > sudo apt install ecj # version 3.16.0-1
> 
> It means that you are missing java compiler that is responsible for running local maven builds.
> 
> Can you share the output of `java -version `?

openjdk version "1.8.0_292"
OpenJDK Runtime Environment (build 1.8.0_292-8u292-b10-0ubuntu1~20.04-b10)
OpenJDK 64-Bit Server VM (build 25.292-b10, mixed mode)

**Comment by kdaud** [comment]
@mherman22 are you sure you put the right command when checking the compiler version? Retry the command

` javac  -version`

**Comment by mherman22** [comment]
> @mherman22 are you sure you put the right command when checking the compiler version? Retry the command
> 
> ` javac -version`

`mherman22@mherman22:~$ javac -version

Command 'javac' not found, but can be installed with:

sudo apt install openjdk-11-jdk-headless  # version 11.0.11+9-0ubuntu2~20.04, or
sudo apt install default-jdk              # version 2:1.11-72
sudo apt install openjdk-16-jdk-headless  # version 16.0.1+9-1~20.04
sudo apt install openjdk-8-jdk-headless   # version 8u292-b10-0ubuntu1~20.04
sudo apt install openjdk-13-jdk-headless  # version 13.0.7+5-0ubuntu1~20.04
sudo apt install ecj                      # version 3.16.0-1`

**Comment by kdaud** [comment]
Share the output of `mvn  -v`

**Comment by mherman22** [comment]
> Share the output of `mvn -v`

```
mherman22@mherman22:~$ mvn -v
Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 1.8.0_292, vendor: Private Build, runtime: /usr/lib/jvm/java-8-openjdk-amd64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.11.0-27-generic", arch: "amd64", family: "unix"
mherman22@mherman22:~$
```

**Comment by kdaud** [comment]
run 

`sudo rm -rf .m2/repository/org/openmrs/distro`

And then run the `mvn clean install ` against the module again and share the experience

**Comment by mherman22** [comment]
> run
> 
> `sudo rm -rf .m2/repository/org/openmrs/distro`
> 
> And then run the `mvn clean install ` against the module again and share the experience

the same result shown above is maintained

**Comment by kdaud** [comment]
>the same result shown above is maintained

@mherman22 lets catch up on Sunday or Monday and we resolve the snag !

**Comment by mherman22** [comment]
> > the same result shown above is maintained
> 
> @mherman22 lets catch up on Sunday or Monday and we resolve the snag !

oh great, happy sabbath

**Comment by mherman22** [comment]
I had a call with @kdaud and we were able to resolve the blocker I had when I ran `mvn clean install` against distro anduitestframework. 

The error here https://github.com/openmrs/openmrs-distro-referenceapplication/pull/555#issuecomment-912570001 clearly indicated that I lacked a java compiler which we have installed and the builds were successful

**Review by kdaud — APPROVED** [review]
Its high time this commit entered into master !

cc: @dkayiwa

**Pull request merged**